packages/nimble-components/src/anchor-menu-item/tests/anchor.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { html } from '@microsoft/fast-element';
```

Comment:
this file would be called anchor-menu-item.spec.ts to align with conventions

Reply:
Fixed.